### PR TITLE
#955: right-aligned actions menu on workpackages and change requests,…

### LINKED ISSUE #955: Right align actions menu and line above delete
### Description

We want the actions menu on projects, workpackages, and change requests to be right aligned with the actions button. We also want a line that seperates the delete button with the other buttons. There should also be a trash icon for the delete like there are icons for the other options.

### Acceptance Criteria

^

### Proposed Solution

View the Change request delete for what the line should look like, but I believe it should be right aligned not left aligned as it is right now. So right align all of them and make sure they all have a line seperating delete from the other options

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by stephanie-xu**
## Changes

Trash icon added to change requests page, added line between delete and other buttons in actions dropdown menus (on project page and workpackages page) and rightaligned dropdown menus for work packages and change requests page.

## Notes
Changed actions menu, which is also used by reimbursment request actions. Screenshots attached for new behavior of its dropdown menu (only change is right-alignment of actions menu).

## Screenshots
Screenshot 2023-09-27 at 8 41 40 PM
Screenshot 2023-09-27 at 8 42 03 PM
Screenshot 2023-09-27 at 8 42 23 PM
Screenshot 2023-09-27 at 8 42 45 PM
Screenshot 2023-09-27 at 8 43 34 PM
Screenshot 2023-09-27 at 8 43 50 PM
Reimbursement request details:
Screenshot 2023-09-27 at 9 06 59 PM
Screenshot 2023-09-27 at 9 07 29 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #955

### CODE REVIEW

**Review by tigerhippo — COMMENTED** [review]
This looks great Stephanie, awesome job! I'm gonna check in with Reid about a couple potential fixes, but you're good for now.

**Review by ryanhowe28 — APPROVED** [review]
Looks great!

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
       : 'Assign to My Team';
 
     return (
-      <MenuItem onClick={handleAssignToMyTeam}>
+      <MenuItem onClick={handleAssignToMyTeam} divider={true}>
```
Comment (RChandler234) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:114]: if the user isn't a head of a team, it doesn't show the divider correctly

**Review by RChandler234 — CHANGES_REQUESTED** [review]
You should also try to switch ProjectViewContainer and WorkPackageViewContainer to use ActionsMenu instead of what they currently use

**Pull request closed without merging**